__tests__/app.test.js

```diff
@@ -62,8 +62,46 @@ describe('App', () => {
                 .get('/api/reviews')
                 .expect(200)
                 .then(({body}) => {
-                    expect(body).toBeSortedBy("created_at", {descending: true})
+                    expect(body.reviews).toBeSortedBy("created_at", {descending: true})
                 })
         });
     });
+    describe('GET /api/reviews/:review_id', () => {
+        it('should return a review object by review id', () => {
+            return request(app)
+            .get('/api/reviews/1')
+            .then(({body}) => {
+                expect(body).toEqual({review: {
```

Comment:
A nice test, though it could be made a little more flexible by using the objectContaining matcher (paired with your test below checking the id is correct) - this would mean that if you ever changed your review objects (eg added a new property), this test would still pass.